### PR TITLE
Update Setup_and_Flashing_Guide.md

### PR DESCRIPTION
**Pull request by JaY-III**
I think this more inline with the other documentation. 
It keeps all instructions in a single location and I have added notes and examples.

### CODE REVIEW

**Comment by JaY-III** [comment]
Ever have a chance to look at this?

File: V0_Display/Documentation/Setup_and_Flashing_Guide.md [hunk 1]
```diff
@@ -16,38 +18,89 @@
...
+	#   Adjust this value if you get some vertical stripes on your display. (31 seems to be a good value)
+	
+	#   RGB Values should be set to match your printer colour scheme
+	[neopixel displayStatus]
+	pin: displayEncoder:PA0
+	chain_count: 1
+	color_order: GRB
+	initial_RED: 0.01
+	initial_GREEN: 0
+	initial_BLUE: 0.01
```
Comment (timmit99) [inline V0_Display/Documentation/Setup_and_Flashing_Guide.md:102]: I would remove this and direct the user to either UPLOAD or COPY the contents of the config file in the GitHub so if i make any changes i dont have to change it in two places.

13) Download the (link to CFG) file and upload that to the same folder as your printer.cfg.

File: V0_Display/Documentation/Setup_and_Flashing_Guide.md [hunk 2]
```diff
@@ -6,7 +6,9 @@
 
 	a) If you have a pre-flashed board:
 		-Install the boot jumper and reset the board. This will put the board into DFU mode.
-	
+		
+		This is required for the LDO Display
+
```
Comment (timmit99) [inline V0_Display/Documentation/Setup_and_Flashing_Guide.md:11]: this is not unique to the LDO displays Best to ASSUME the board is already flashed and explicitly tell the user to install the jumper.

File: V0_Display/Documentation/Setup_and_Flashing_Guide.md [hunk 3]
```diff
@@ -6,7 +6,9 @@
 
 	a) If you have a pre-flashed board:
 		-Install the boot jumper and reset the board. This will put the board into DFU mode.
-	
+		
+		This is required for the LDO Display
+		
 	b) If you have an unflashed board:
 		- The MCU default's to DFU mode if there is no other firmware installed.
```
Comment (timmit99) [inline V0_Display/Documentation/Setup_and_Flashing_Guide.md:14]: Remove these lines

**Review by timmit99 — COMMENTED** [review]
Made some notes. i like the direction this is going though.

**Comment by timmit99** [comment]
> Ever have a chance to look at this?

sorry for the delay!

**Comment by raymondh2** [comment]
Closing due to inactivity

**Pull request closed without merging**